src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,38 @@
+import {Dispatch, FormEvent, SetStateAction} from "react";
+import {useTranslation} from "react-i18next";
+import classNames from "classnames";
+import {ReactComponent as SearchIcon} from "assets/icons/search.svg";
+import {ReactComponent as ClearIcon} from "assets/icons/close.svg";
+import "./SearchBar.scss";
+
+type SearchBarProps = {
+  className?: string;
+  disabled?: boolean;
+  value: string;
+  handleValueChange: Dispatch<SetStateAction<string>>;
+};
+
+/*
+ * stateless search bar component.
+ * if the input is not empty, it's clearable using the X button
+ */
+export const SearchBar = (props: SearchBarProps) => {
+  const {t} = useTranslation();
+
+  const updateInput = (e: FormEvent<HTMLInputElement>) => props.handleValueChange(e.currentTarget.value);
+  const clearInput = () => props.handleValueChange("");
+
+  return (
+    <div className={classNames(props.className, "search-bar", {"search-bar--disabled": props.disabled})}>
```

Comment:
Clicking on the search bar should always focus the input. Right now, the input is only focused if you actually click on the input within the container.